src/gigs-board/entity/community/header.jsx

```diff
@@ -172,7 +172,7 @@ const CommunityHeader = ({ activeTabTitle, handle }) => {
       title: "GitHub",
     },
 
-    ...(communityData.telegram_handle !== null
+    ...((value?.length ?? 0) > 0
```

Comment:
Seems a typo? I don't see any `value` in this file

Reply:
Yes, it is, thanks